doc/deploy/android.md

```diff
@@ -1,33 +1,129 @@
 
+### Node
+  1. Install node: `$brew install node`.
+  1. Then use it actually install our packages:`$npm install` (You probably
+  already did this when doing work on the application, but it can't hurt to
+  update if you pulled from master)
+
+## Ready to build?
+  1. Open `config.xml`: On line 2, increment `android-versionCode`
+  with your appropriately chosen new version number.
+  1. You may make a debug build first to try the changes to your application on
+  an emulator or phone: `$ionic cordova build android`.
+  1. If you have an Android: Plug it into the computer with a USB cable and enable USB
+  debugging. Side-load like so, replacing the paths with your own:
+ `~/Library/Android/sdk/platform-tools/adb install /Users/my_name/pvta-multiplatform/platforms/android/build/outputs/apk/debug/android-debug.apk
+  1. Obtain the PVTrAck signing key. See [the keys doc](android-keys.md) for more info.
+  Place it in the `platforms/android` directory. The `jks` extension is gitignored by default.
+  1. Obtain the password. See Molly, Adam, or Karin for access.
+  1. `$cp keystore.properties.example platforms/android/keystore.properties`
+  1. Replace the two password lines in `platforms/android/keystore.properties` with the actual
+  password. No space between the `=` sign and the password! This file is also gitignored. Please
+  do not put the real password in the `.example` file and push it to github.
+  1. Now you can build a release APK that should be signed automatically:
+  `$ionic cordova build android --prod --release`
+The terminal output should tell you your release location. If it's something like this:
+`/platforms/android/build/outputs/apk/release/android-release.apk`
+you should be fine.
+"android-release" and not "android-release-unsigned."
+that's a pretty good indication that you do not have a signed release APK.
+  1. In the [Google Play Developer Console](play.google.com/apps/publish), follow the instructions for uploading a new APK.
+  1. 🎉
+
+## Troubleshooting
+
+  - **You get an error related to whether this is an Android project**
+    
+    Try adding and removing the platform: `$ionic platform rm android`,`$ionic platform add android`
+
+  - **Your error message occurred when running `$cordova`**
+  
+  Try running`$cordova requirements`.  Did you meet all of them? 
+  Did you miss a step? If you're missing IOS requirements, that's OK until
+  you decide to build the IOS app--you don't need any of them for Android.
+
+  - **Error: Failed to find 'ANDROID_HOME' or 'JAVA_HOME' environment variable. Try setting it manually.**
+  
+  For "ANDROID_HOME" issues, find the location of your sdk tools.
+  Android studio should have told you where they were installed. For example,
+  my location was `/Users/my_name/Library/Android/sdk`
+
+  For "JAVA_HOME" problems, find the location of your JDK installation. My location
+  was `/Library/Java/JavaVirtualMachines/jdk-11.jdk/Contents/Home` and I don't know how
```

Comment:
Are you sure it's "`jdk-11`" Your instructions below seem to imply that you should be using Java 8 not Java 11.

Reply:
Or, you're saying that's what is _was_ but not what it should have been?

Reply:
Woops I copied and pasted from a "failed examples" part of my running documentation. Fuxed